mapzen-places-api/src/main/java/com/mapzen/places/api/internal/PlacePickerActivity.java

```diff
@@ -46,7 +47,9 @@
     autocompleteView.setActivity(this);
 
     //TODO inject
-    presenter = new PlacePickerPresenterImpl(new PeliasPlaceDetailFetcher());
+    Pelias pelias = new Pelias();
```

Comment:
The `Pelias` instance passed into the `PeliasPlaceDetailFetcher` needs to be configured to use the Mapzen API key.